pkg/providers/vault/tests/main.go

```diff
@@ -0,0 +1,169 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	"github.com/edgexfoundry/go-mod-secrets/pkg/providers/vault"
+)
+
+var rootTokenPtr = flag.String("token", "s.KhQILbtzYTGKbFQtonjuqNEQ", "token string (should be root token)")
+var timeWindowPrt = flag.Int("window", 15, "how long to run the test in seconds")
+var tokenPeriodPtr = flag.Int("period", 10, "token period in seconds")
+var vaultHostPtr = flag.String("host", "localhost", "host of vault server")
+
+func init() {
+	// example with short version for long flag
+	flag.StringVar(rootTokenPtr, "t", "s.KhQILbtzYTGKbFQtonjuqNEQ", "token string (should be root token)")
+	flag.IntVar(timeWindowPrt, "w", 15, "how long to run the test in seconds")
+	flag.IntVar(tokenPeriodPtr, "p", 10, "token period in seconds")
+	flag.StringVar(vaultHostPtr, "h", "localhost", "host of vault server")
+}
+
+func main() {
+	//Read root token
+	flag.Parse()
+	rootToken := *rootTokenPtr
+	totalTimeWindow := time.Duration(*timeWindowPrt) * time.Second
+	tokenPeriod := time.Duration(*tokenPeriodPtr) * time.Second
+	vaultHost := *vaultHostPtr
+
+	logger := logger.NewClientStdOut("test", false, "DEBUG")
+
+	// Create token
+	vaultPort := 8200
+	tokenCreatPath := "/v1/auth/token/create"
```

Comment:
Better to use complete word tokenCreatePath VS. tokenCreatPath for search purpose.

Reply:
ok